File: Objektorientierung/VerketteteListenRuby/verkettetesElement.rb
class VerkettetesElement
    def initialize(w)
        @wert = w
        @voergaenger = nil
        @nachfolger = nil
    end

    def vorgaenger=(v)
        @vorgaenger = v
    end

    def nachfolger=(n)
        @nachfolger = n
    end

    def wert=(w)
        @wert = w
    end

    def vorgaenger()
        return @vorgaenger
    end

    def nachfolger()
        return @nachfolger
    end

    def wert()
        return @wert
    end
end

File: Objektorientierung/VerketteteListenPython/verkettetesElement.py
class VerkettetesElement:

    def __init__(self,w):
        self.wert = w
        self.voergaenger = None
        self.nachfolger = None

    def setVorgaenger(self,v):
        self.vorgaenger = v

    def setNachfolger(self,n):
        self.nachfolger = n
    
    def setWert(self,w):
        self.wert = w

    def getVorgaenger(self):
        return self.vorgaenger

    def getNachfolger(self):
        return self.nachfolger

    def getWert(self):
        return self.wert
    

